### PR TITLE
Sasha/report macs

### PR DESCRIPTION
**Pull request by sashas-lb**
report mac addresses from inaugurator during checkin.
All network devices mac addresses are reported during checkin operation as hwinfo['net'] in checkin message.

### CODE REVIEW

**Comment by sashas-lb** [comment]
verified that after kexec in from inaugurator devices are all reported and link is detected

File: Makefile.lb [hunk 1]
```diff
@@ -8,6 +8,9 @@ all: build
 build:
 	$(Q)$(MAKE) -f $(inaugurator_makefile_dir)/Makefile $@ BUILD_HOST=docker
 
+docker-shell:
+	$(Q)$(MAKE) -f $(inaugurator_makefile_dir)/Makefile $@ BUILD_HOST=docker
```
Comment (adir-lb) [inline Makefile.lb]: what is the difference between this target and `build` target?

File: inaugurator/network.py [hunk 2]
```diff
@@ -35,3 +36,13 @@ def _interfacesTable(self):
         REGEX = re.compile(r'\d+:\s+([^:]+):\s+.*\s+link/ether\s+((?:[a-fA-F0-9]{2}:){5}[a-fA-F0-9]{2})')
         ipOutput = sh.run("/usr/sbin/ip -o link")
         return {mac.lower(): interface for interface, mac in REGEX.findall(ipOutput)}
+
+def list_all_mac_addresses():
+    # network class serial numbers are network devices mac addresses
+    try:
+        cmd="/usr/sbin/lshw -c net 2>/dev/null| /usr/sbin/awk '/serial/{print $2}'"
```
Comment (adir-lb) [inline inaugurator/network.py]: the following command seems simpler - 
```bash
cat /sys/class/net/*/address
```

File: inaugurator/ceremony.py [hunk 3]
```diff
@@ -210,7 +210,8 @@ def _osmosFromNetwork(self, destination):
         if self._args.inauguratorServerAMQPURL:
             self._talkToServer = talktoserver.TalkToServer(
                 amqpURL=self._args.inauguratorServerAMQPURL, myID=self._args.inauguratorMyIDForServer)
-            self._talkToServer.checkIn()
+            hwinfo = {'net' : network.list_all_mac_addresses()}
```
Comment (adir-lb) [inline inaugurator/ceremony.py]: i wonder whether we'd like to return additional data (i.e. speed, vendor).

**Comment by sashas-lb** [comment]
fixed all adirs comment .. @yuval-lb  please re-review and ack .. this had been checked with mlx, intel and solarflare .. seems working

**Pull request merged**